ninja-core/src/test/java/ninja/NinjaDefaultTest.java

```diff
@@ -318,6 +318,11 @@ public void testThatGetInternalServerErrorDoesFallsBackToHtml() throws Exception
     @Test
     public void getInternalServerErrorResult() throws Exception {
         
+        when(ninjaProperties.getWithDefault(
+                Matchers.eq(NinjaConstant.LOCATION_VIEW_HTML_INTERNAL_SERVER_ERROR_KEY), 
+                Matchers.eq(NinjaConstant.LOCATION_VIEW_FTL_HTML_INTERNAL_SERVER_ERROR)))
+                .thenReturn(NinjaConstant.LOCATION_VIEW_FTL_HTML_INTERNAL_SERVER_ERROR);
```

Comment:
can you add a verify at the end of each test that makes sure that getWithDefault was really called?